scripts/count_rosdistro_packages.py

```diff
@@ -14,6 +14,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from __future__ import print_function
```

Comment:
The change has nothing to do with Python 2 / 3 compatibility. The code should work fine with both versions without that line.